xnmt/translator.py

```diff
@@ -191,3 +194,190 @@ def html_report(self, context=None):
     return html
 
 
+class TransformerTranslator(Translator, Serializable, Reportable):
```

Comment:
At some point, I'd like to remove the need for having this be a separate translator, but I think it's OK for now.